config/model_configs/single_column_radiative_equilibrium_gray.yml

```diff
@@ -1,12 +1,13 @@
 dt_save_state_to_disk: "100days"
 initial_condition: "IsothermalProfile"
 hyperdiff: false
-z_elem: 70
+# It seems radiative equilibrium needs a larger dz near the top
+# to be stable, so we use a smaller z_elem
+z_elem: 50
```

Comment:
Should we also use 63 z_elem for uniformity ?

Reply:
See the comments in this yaml. When using 63 this simulation breaks, I think it's because gray radiation is the only temperature tendency in this case, so near the high top it can be a bit stable. I'll change the model top in a separate PR, and we can reevaluate whether this is an issue.